shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -752,7 +752,8 @@ void PlatformView::HandlePlatformMessage(
   if (response && !response_handled) {
     // response_handled should be true if the response was completed.
     FML_DCHECK(!response->is_complete());
-    response->CompleteEmpty();
+    response->Complete(std::make_unique<fml::DataMapping>(
+        std::vector<uint8_t>({'[', '0', ']'})));
```

Comment:
Return this value from the specific method implementations that need it.

Unimplemented messages should call `CompleteEmpty` to indicate that they were not handled.

Reply:
+1 to what jason says here.

We need to audit our method implementations and  call this in each one of those places, leaving CompleteEmpty() in for the cases we truly didn't handle.